roles/postgresql/defaults/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+postgresql_image: docker.io/library/postgres:16
+postgresql_container_name: postgresql
+postgresql_network: host
+postgresql_restart_policy: always
+
+postgresql_data_dir: /var/lib/pgsql/data
+postgresql_volumes:
+  - "{{ postgresql_data_dir }}:/var/lib/postgresql/data:Z"
+  - "/etc/postgresql/:/etc/postgresql:Z"
+
+postgresql_admin_user: postgres
+postgresql_admin_password: "CHANGEME"
+postgresql_port: 5432
+
+postgresql_databases:
```

Comment:
I think we should define this at the playbook level, or a wrapper role rather than hardcoding this role to knowledge of foreman/candlepin/pulp.